src/sdam/server.ts

```diff
@@ -420,6 +420,20 @@ export class Server extends TypedEventEmitter<ServerEvents> {
       }
     }
   }
+
+  /**
+   * Decrement the operation count, returning the new count.
+   */
+  private decrementOperationCount(): number {
+    return (this.s.operationCount -= 1);
+  }
+
+  /**
+   * Increment the operation count, returning the new count.
+   */
+  private incrementOperationCount(): number {
+    return (this.s.operationCount += 1);
+  }
```

Comment:
Any reason to return the new count, if we're not chaining it or using the returned value?

Reply:
No was just a general practice I like to use - if the method would normally be void try potentially returning something useful if possible.